src/commands/deploy.ts

```diff
@@ -154,38 +131,46 @@ export async function deploy(context: IDeployWizardContext, confirmDeployment: b
         const warning: string = `Are you sure you want to deploy to "${node.root.client.fullName}"? This will overwrite any previous deployment and cannot be undone.`;
         context.telemetry.properties.cancelStep = 'confirmDestructiveDeployment';
         const items: MessageItem[] = [{ title: 'Deploy' }];
+
         const resetDefault: MessageItem = { title: 'Reset default' };
-        if (defaultWebAppToDeploy) {
+        if (context.deployedWithConfigs) {
             items.push(resetDefault);
         }
+
         items.push(DialogResponses.cancel);
 
         // a temporary workaround for this issue: https://github.com/Microsoft/vscode-azureappservice/issues/844
         await delay(500);
         const result: MessageItem = await ext.ui.showWarningMessage(warning, { modal: true }, ...items);
         if (result === resetDefault) {
-            const localRootPath = nonNullValue(currentWorkspace).uri.fsPath;
-            const settingsPath = path.join(localRootPath, '.vscode', 'settings.json');
-            const doc = await workspace.openTextDocument(Uri.file(settingsPath));
-            window.showTextDocument(doc);
-            await workspaceConfig.update(constants.configurationSettings.defaultWebAppToDeploy, '');
+            if (context.configurationTarget === ConfigurationTarget.Global) {
+                await commands.executeCommand('workbench.action.openGlobalSettings');
+            } else {
+                const localRootPath = nonNullValue(currentWorkspace).uri.fsPath;
```

Comment:
Can you change this case to use a command as well? `'workbench.action.openSettings'` maybe?

Reply:
Using `workbench.action.openSettingsJson` seems to just open the User settings as well whether I have a single root or multiroot workspace 😕